src/main/java/org/jabref/gui/util/BaseDialog.java

```diff
@@ -20,7 +20,7 @@ protected BaseDialog() {
         });
 
         setDialogIcon(IconTheme.getJabRefImageFX());
-
+        setResizable(true);
```

Comment:
It's a good idea to set this globally. Can you please remove the other explicit `setResizable` calls that are now no longer needed.

Reply:
I removed it everywhere except in the prefs dialog to not create a conflict with your other PR